### ISSUE TITLE
value of threshold

### ISSUE THREAD

**Issue by JohnBr90** [hellloxiaotian/PSDNet#9]
In the paper it is written that the value of threshold of wide enhancement block is 0.75 but in the dsrnet.py in line 168 ( if decision[:, 1].cpu().detach().numpy().all() == 1: ) , threshold is set to 1 so what is its value?

**Comment by XuanyuZhang** [comment]
Thank you for your attention!
From our paper you can see that when this value is set to 0.75, the results get the best PSNR. Therefore, I suggest setting it to 0.75. 
You can also change this value as you need.

**Comment by XuanyuZhang** [comment]
> In the paper it is written that the value of threshold of wide enhancement block is 0.75 but in the dsrnet.py in line 168 ( if decision[:, 1].cpu().detach().numpy().all() == 1: ) , threshold is set to 1 so what is its value?

Thank you for your attention!
From our paper you can see that when this value is set to 0.75, the results get the best PSNR. Therefore, I suggest setting it to 0.75.
You can also change this value as you need.
